### ISSUE TITLE
generate google cloud dns config

### ISSUE THREAD

**Issue by awesomescot** [carlpett/tfz53#23]
It would be great if this script could also generate a terraform config for other providers.  I'm looking for something to generate a google cloud dns config file, but there's nothing out there.

Thanks so much for the great script!

**Comment by carlpett** [comment]
Yes, I agree. I've been planning to add some support for multiple providers, but haven't had the time unfortunately. 
In the meantime, #22 was filed as a patch that completely replaces the Route53 support with GCP instead. If you're up for building the binary yourself, you can check that out!
